dapp/pages/history.js

```diff
@@ -8,18 +8,42 @@ import BalanceHeader from 'components/buySell/BalanceHeader'
 import TransactionHistory from 'components/TransactionHistory'
 import GetOUSD from 'components/GetOUSD'
 import { assetRootPath } from 'utils/image'
+import { useOverrideAccount } from '../src/utils/hooks'
+import ErrorModal from '../src/components/buySell/ErrorModal'
 
 export default function History({ locale, onLocale }) {
   const { active } = useWeb3React()
+  const { overrideAccount, isValid } = useOverrideAccount()
+  const [showErrorModal, setShowErrorModal] = useState(!isValid)
 
+  const errorMap = [
```

Comment:
All values on this object seem to be independent of the component's props. Let's move it outside the component?

Reply:
sure